src/main/groovy/az/pashabank/service/impl/StudentServiceImpl.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Created By Nadir Jabbarli
+ * Date : 7/5/2022
+ * Time : 11:04 AM
+ * Project Name : student-service
+ */
+
+package az.pashabank.service.impl;
+
+import az.pashabank.mapper.StudentMapper;
+import az.pashabank.model.entity.StudentEntity;
+import az.pashabank.model.dto.StudentDto;
+import az.pashabank.model.dto.StudentRequestDto;
+import az.pashabank.repository.StudentRepository;
+import az.pashabank.service.StudentService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class StudentServiceImpl implements StudentService {
+
+    final String DOMAIN = "@pashabank.az";
+
+    private final StudentMapper studentMapper;
+
+    private final StudentRepository studentRepository;
+
+    public StudentServiceImpl(StudentRepository studentRepository) {
+        this.studentRepository = studentRepository;
+        studentMapper = StudentMapper.INSTANCE;
+    }
+
+    @Override
+    public String generateStudentEmailAccount(Long id, String name, String surname) {
+        String firstLetter = String.valueOf(name.charAt(0));
+        String email = firstLetter.concat(surname).concat(String.valueOf(id));
+
+        return email.concat(DOMAIN);
```

Comment:
let's do this with just one line:

return String.valueOf(name.charAt(0)).concat(surname).concat(String.valueOf(id)).concat(DOMAIN);